src/main/webapp/WEB-INF/tags/instructor/studentsList/filterCourse.tag

```diff
@@ -0,0 +1,11 @@
+<%@ tag description="instructorStudentList - Contents for filter box per course" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ taglib tagdir="/WEB-INF/tags/instructor/studentsList" prefix="tisl" %>
+<%@ attribute name="data" type="teammates.ui.template.InstructorStudentsListFilterCourses" required="true" %>
+<%@ attribute name="index" required="true" %>
+<div class="checkbox">
+    <input id="course_check-${index}" type="checkbox">
+    <label for="course_check-${index}">
+        [${data.courseId}] : ${data.courseName}
+    </label>
+</div>
```

Comment:
Will this be used anywhere else in the future?
If not, combining it with filterBox.tag should be fine too.